llvm/test/CodeGen/AMDGPU/adjust-writemask-cse.ll

```diff
@@ -0,0 +1,32 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 3
+; RUN: llc -march=amdgcn -mcpu=gfx1030 -stop-after=finalize-isel < %s | FileCheck -check-prefix=GFX10
```

Comment:
Missing %s input to FileCheck?

Reply:
Whoops. update_mir_test_checks.py tolerated it and I forgot to re-run lit.